tools/api-markdown-documenter/README.md

```diff
@@ -56,6 +56,62 @@ To access the `beta` APIs, import via `@fluid-tools/api-markdown-documenter/beta
 
 <!-- AUTO-GENERATED-CONTENT:END -->
 
+## Architecture
```

Comment:
Moved this section up, as it is contextually important when enumerating the APIs.

Reply:
I'm probably going to do a follow-up PR to re-organize the contents of this document some. This library has evolved enough, that I think it needs to flow differently.